packages/backend/src/clients/github/Github.ts

```diff
@@ -49,7 +51,7 @@ export const getOctokitRestForUser = (authToken: string) => {
     };
 };
 
-export const getOctokitRestForApp = (installationId: string) => {
+export const getOctokitRestForApp = (installationId: string): OctokitRest => {
```

Comment:
this changed because we need to add now return types everywhere ? is this mandatory ?